vm/generator/main/generator.cc

```diff
@@ -105,9 +105,11 @@ int main(int argc, char* argv[]) {
   }
 
   // Parse source file
-  ASTUnit *unit = ASTUnit::LoadFromASTFile(astFile,
-                                           Diags, FileSystemOpts,
-                                           false, 0, 0, true);
+  clang::CompilerInstance CI;
+  std::unique_ptr<ASTUnit> unit = ASTUnit::LoadFromASTFile(astFile,
+                                           PCHContainerOperations().getRawReader(),
```

Comment:
Wrapped arguments should align with the the first argument. However, here it becomes very long, so maybe wrap directly after `(`, in which case indent 2 (or 4? I don't remember, best check somewhere else we did this) spaces wrt the `s` of `std::`.